inc/commondbconnexity.class.php

```diff
@@ -189,15 +189,44 @@ static function getItemsAssociatedTo($itemtype, $items_id) {
     */
    static function getItemsAssociationRequest($itemtype, $items_id) {
       global $DB;
-
-      return $DB->request([
+      return $DB->request(static::getSQLCriteriaToSearchForItem($itemtype, $items_id));
+      /*return $DB->request([
          'SELECT' => 'id',
          'FROM'   => static::getTable(),
          'WHERE'  => [
             'itemtype'  => $itemtype,
             'items_id'  => $items_id
          ]
-      ]);
+      ]);*/
+   }
+
+   public static final function getItemField($itemtype) {
```

Comment:
I would have split this method between `CommonDBChild` and `CommonDBRelation` (as it is done for the `getSQLCriteriaToSearchForItem` method), and may be declare it here as abstract.